components/midea_ac_lan

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Midea AC on LAN",
+  "owner": ["@wyapx"],
+  "manifest": "https://github.com/wyapx/hass_midea_acc/raw/master/custom_components/midea_lan/manifest.json",
```

Comment:
This link is a redirect, use the target for that URL instead.

Reply:
Okay, I will make change